### ISSUE TITLE
Auto Update Broken

### ISSUE THREAD

**Issue by Ninjistix** [Steveice10/FBI#467]
Not sure if the recent version fixed this but doing a update from version 2.5.0 results in

```
Error
Failed to check for update
Result Code: 0xD963F806
Level: Permanent (27)
Summary: Internal (11)
Module: Application (254)
Desc: Bad Data (6)
```

**Comment by Ninjistix** [comment]
sorry i had missed version 2.5.1 (will try installing that version and retesting the auto update)

**Comment by Steveice10** [comment]
First issue was fixed in 2.5.1, second was fixed in 2.5.2

**Comment by Ninjistix** [comment]
okay thanks a lot, just wanted to make a report (in case)
